storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/values/GroupValue.java

```diff
@@ -43,7 +43,7 @@
      * Creator method.
      * @param action
      */
-    public static final GroupValue create(ActionType action, boolean onlyArtifacts) {
+    public static final GroupValue create(ActionType action, boolean onlyArtifacts /* <-- TODO What is this? */) {
```

Comment:
it's used for operations like, delete all artifacts in a group but keeping the group